### PR TITLE
docs: add dragonfly to integration.md

### PR DESCRIPTION
**Pull request by gaius-qi**
Signed-off-by: Gaius <gaius.qi@gmail.com>

### CODE REVIEW

**Comment by gaius-qi** [comment]
Fixes #966 

@ktock

File: docs/overview.md [hunk 1]
```diff
@@ -189,6 +189,10 @@ insecure = true
 
 The config file can be passed to stargz snapshotter using `containerd-stargz-grpc`'s `--config` option.
 
+If the kubernetes cluster is very large, it is recommended to use [dragonfly](https://d7y.io)
+as registry mirror to reduce the requests and traffic of the origin based on P2P,
+[please refer to the doc](https://d7y.io/docs/setup/integration/stargz/).
```
Comment (ktock) [inline docs/overview.md:194]: We should have this in a separated documentation like `integration.md`.

Comment (gaius-qi) [reply docs/overview.md]: I have added `integrations.md`.

**Comment by gaius-qi** [comment]
@ktock

**Comment by ktock** [comment]
The link will be added through  #995.

**Pull request closed without merging**